### PR TITLE
LPS-87060 Update the expression when deleting a field

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/metal/js/components/RuleEditor/RuleEditor.es.js [hunk 1]
```diff
@@ -883,6 +883,35 @@ describe(
...
+
+								const formBuilderFields = component.fieldOptions.map(a => a.fieldName);
+
+								component.pages = [];
+
+								jest.runAllTimers();
+
+								const notEquals = formBuilderFields.some(
+									item => {
+										return component.deletedFields.indexOf(item) == -1;
```
Comment (clovisdasilvaneto) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/metal/js/components/RuleEditor/RuleEditor.es.js]: You don't need brackets, you could use item => component.deletedFields.indexOf(item) == -1;

**Comment by alexcandidos** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by alinedoleron** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by alinedoleron** [comment]
resent to https://github.com/clovisdasilvaneto/liferay-portal/pull/76

**Pull request closed without merging**